eth/tracers/native/call.go

```diff
@@ -130,10 +139,37 @@ func (t *callTracer) CaptureEnd(output []byte, gasUsed uint64, _ time.Duration,
 
 // CaptureState implements the EVMLogger interface to trace a single step of VM execution.
 func (t *callTracer) CaptureState(pc uint64, op vm.OpCode, gas, cost uint64, scope *vm.ScopeContext, rData []byte, depth int, err error) {
+	if !t.config.WithLog {
```

Comment:
In case `onlyTopCall` is true we shouldn't parse logs for nested calls. So `onlyTopCall == true && depth > 0: return`

Reply:
thanks, fixed